source/postprocess/particles.cc

```diff
@@ -559,30 +597,46 @@ namespace aspect
     void
     Particles<dim>::save (std::map<std::string, std::string> &status_strings) const
     {
-      std::ostringstream os;
-      aspect::oarchive oa (os);
+      for (unsigned int particle_world = 0; particle_world < this->n_particle_worlds(); ++particle_world)
+        {
+          std::string particles_output_base_name = "Particles";
+          if (particle_world > 0)
```

Comment:
For load/save we don't need to be backwards compatible, right? We could always put the number in the filename.

Reply:
I will make a follow-up that adds the number of particle worlds to the list of critical model parameters. This is a parameter that should most likely not be changed across model restarts (because it will change the amount of data attached to the triangulation checkpoint).